tests/test_model_manual.py

```diff
@@ -9,8 +9,13 @@
 from pint.models.astrometry import AstrometryEquatorial
 from pint.models.dispersion_model import DispersionDM, DispersionDMX
 from pint.models.spindown import Spindown
-from pint.models.model_builder import UnknownBinaryModel, get_model, get_model_new
-from pint.models.timing_model import MissingParameter, TimingModel, Component
+from pint.models.model_builder import get_model, get_model_new
```

Comment:
Since this replaces both the existing `get_model` and the broken `get_model_new`, please delete `get_model_new`. This will necessitate better tests, but testing your new code against broken code isn't helpful.